gravitee-am-service/src/main/java/io/gravitee/am/service/impl/EntrypointServiceImpl.java

```diff
@@ -60,15 +61,18 @@ public class EntrypointServiceImpl implements EntrypointService {
     private final OrganizationService organizationService;
     private final AuditService auditService;
     private final VirtualHostValidator virtualHostValidator;
+    private final DomainService domainService;
 
     public EntrypointServiceImpl(@Lazy EntrypointRepository entrypointRepository,
                                  @Lazy OrganizationService organizationService,
                                  AuditService auditService,
-                                 VirtualHostValidator virtualHostValidator) {
+                                 VirtualHostValidator virtualHostValidator,
+                                 DomainService domainService) {
```

Comment:
**Thought**: Injection DomainService into EmailService looks stranve to me.

Reply:
In EntrypointServiceImpl it has to be injected

Reply:
ok